robot/components/drive.py

```diff
@@ -0,0 +1,41 @@
+import wpilib
+
+from networktables import NetworkTable
+from networktables.util import ntproperty
+
+ENCODER_ROTATION = 1023
+WHEEL_DIAMETER = 7.639
+
+class Drive:
+    robot_drive = wpilib.RobotDrive
+    sd = NetworkTable
+
+    def __init__(self):
+        self.sd = NetworkTable.getTable('/SmartDashboard')
```

Comment:
since tunables will automatically put stuff in /components/drive, I recommend doing the same for your own values